package.json

```diff
@@ -76,12 +76,13 @@
     "@types/resolve": "0.0.8",
     "@types/semver": "6.0.0",
     "@types/tmp": "0.1.0",
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
     "babel-eslint": "10.0.1",
```

Comment:
If we're using `@typescript-eslint/parser` everywhere now, can we drop `babel-eslint` from our devDeps?

Reply:
Oh, good call – I think so, yes. I'll do that and confirm everything still works.